psycop/common/sequence_models/tests/test_train.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+import pytest
+
+from psycop.common.sequence_models.dataset import PatientSliceDataset
+from psycop.common.sequence_models.registry import Registry
+from psycop.common.sequence_models.train import train
+
+from .utils import create_patients
+
+
+@pytest.fixture()
+def config_path() -> Path:
+    return Path(__file__).parent / "test_config.cfg"
+
+
+@Registry.datasets.register("test_dataset")
+def create_test_dataset() -> PatientSliceDataset:
+    patients = create_patients()
+    return PatientSliceDataset(patient_slices=[p.as_slice() for p in patients])
+
+
+def test_train(config_path: Path):
+    """test of the train function"""
+    train(config_path)
```

Comment:
this is an example of the application script. All it needs is a path to a config